docs/src/main/asciidoc/deploying-to-openshift-native-howto.adoc

```diff
@@ -15,17 +14,24 @@ include::_attributes.adoc[]
 
 You can deploy your native {project-name} applications to {openshift} compiled to native executables by using the Docker build strategy.
 
-You must create a native executable for your application that targets the Linux AMD64 operating system.
-If your host operating system is different from this, create a native Linux executable by using a container runtime, for example, Docker or Podman.
+You must create a native executable for your application that targets a supported operating system.
+The target operating system must match the architecture of the operating system where you create the native executable.
+If they do not, you create a native Linux executable by using a container runtime, for example, Docker or Podman.
```

Comment:
I am not sure if this is true. It should be possible to create native executable for multiple architectures, but I don't think it is possible for example to create Aarch64 native executable on `x86_64` executor using Podman, e.g. `podman build --platform linux/amd64,linux/arm64` but can I use Quarkus native builder image on `x86_64` executor to build Aarch64 executable using Podman?

I think @iocanel knows more than I do about this. Please let me know.

Reply:
@maxandersen, might you be able to help clarify this question? Many thanks in advance.

Reply:
Correct - if the archtiecture does not match you will also hit issues; but i'm not sure its worth highlighting all the details here because it is doable in some cases.

I would just keep it simple and show the common case:

```
You must create a native executable for your application that targets a supported operating system and match the architecture.
This means if you are buidling Windows, you create a native Linux executable by using a container runtime, for example, Docker or Podman.
```

Reply:
Updated per Max's proposal: